website/src/components/Header/Header.tsx

```diff
@@ -54,75 +53,67 @@ function AccountButton() {
 }
 
 export function Header(props) {
-  const { colorMode } = useColorMode();
-  const borderClass = props.transparent
```

Comment:
This borderClass stuff is important so that the header can render transparent properly without a border (signed out homepage)